src/QsCompiler/Tests.Compiler/TestCases/TypeChecking.qs

```diff
@@ -579,5 +579,30 @@ namespace Microsoft.Quantum.Testing.TypeChecking {
         let fct = PartialApplication29();
         fct(1);
     }
-}
 
+    // Sized array constructors
```

Comment:
A test case for the empty array literal `[ ]` would also be good.

Reply:
There are no changes to the behavior of `[]` in this PR. Although it seems like there aren't any existing tests for `[]`.